tests/unit/test_configuration.py

```diff
@@ -43,53 +43,71 @@ def test_default_configuration_structure():
                     else:
-                        assert config_row_noise_level == builder_level.row_noise_level
+                        assert config_row_noise_level == default_row_noise_level
                 if noise_type.token_noise_level:
                     config_token_noise_level = config_level.token_noise_level
                     default_token_noise_level = (
                         DEFAULT_NOISE_VALUES.get(form.name, {})
                         .get("column_noise", {})
                         .get(col.name, {})
                         .get(noise_type.name, {})
-                        .get("token_noise_level", {})
+                        .get("token_noise_level", "no default")
                     )
-                    if default_token_noise_level:
-                        assert config_token_noise_level == default_token_noise_level
+                    if default_token_noise_level == "no default":
+                        assert config_token_noise_level == baseline_level.token_noise_level
                     else:
-                        assert config_token_noise_level == builder_level.token_noise_level
+                        assert config_token_noise_level == default_token_noise_level
                 if noise_type.additional_parameters:
-                    # FIXME: add default check
-
-                    for key, value in noise_type.additional_parameters.items():
-                        config_value = config_level[key]
-                        if isinstance(config_value, float):
-                            assert config_value == value
-                        elif isinstance(config_value, ConfigTree):
-                            assert config_level[key].to_dict() == value
-                        else:
-                            raise AttributeError
+                    config_additional_parameters = {
+                        k: v
+                        for k, v in config_level.to_dict().items()
+                        if k not in ["row_noise_level", "token_noise_level"]
+                    }
+                    default_additional_parameters = (
+                        DEFAULT_NOISE_VALUES.get(form.name, {})
+                        .get("column_noise", {})
+                        .get(col.name, {})
+                        .get(noise_type.name, {})
+                    )
+                    default_additional_parameters = {
+                        k: v
+                        for k, v in default_additional_parameters.items()
+                        if k not in ["row_noise_level", "token_noise_level"]
+                    }
+                    if default_additional_parameters == {}:
+                        assert (
+                            config_additional_parameters
+                            == baseline_level.additional_parameters
+                        )
+                    else:
+                        assert config_additional_parameters == default_additional_parameters
```

Comment:
What if there are multiple additional parameters, and only one of them has an override in the default layer? Then the actual configuration will match neither of these cases. You should compare the keys present in the default layer to the true configuration and the keys not present in the default layer to the baseline_level.